src/org/openstreetmap/josm/gui/tagging/presets/TaggingPreset.java

```diff
@@ -141,6 +133,10 @@ public class TaggingPreset extends AbstractAction implements ActiveLayerChangeLi
      * The types as preparsed collection.
      */
     public transient Set<TaggingPresetType> types;
+    /**
+     * list of regions the preset is applicable for
+     */
+    Set<String> regions;
```

Comment:
```suggestion
    private Set<String> regions;
```

We want to use a `setRegions` here, since it will let us take a string and split it. If this is _not_ private, then the reflection code will try to use this field instead of the `setRegions` method.